src/Event/EventFactory.php

```diff
@@ -45,7 +45,7 @@ public function createFromArray(array $data): Event
     public function createCitizenInitiativeFromArray(array $data): CitizenInitiative
     {
         foreach (['uuid', 'organizer', 'name', 'category', 'description', 'address', 'begin_at', 'finish_at',
-                     'expert_assistance_needed', 'coaching_requested', 'capacity', ] as $key) {
+                     'expert_assistance_needed', 'coaching_requested', 'capacity', 'place', ] as $key) {
```

Comment:
Extra space and comma.

Reply:
Wrong, it's good syntax for StyleCi preset.

Reply:
Yeah but the array has a broken style that makes the parser doing false positive. Ok to leave it as is, not related to your changes anyway.